app/assets/stylesheets/buildout_design_system/components/table.scss

```diff
@@ -1,11 +1,11 @@
 .table {
-  thead th {
-    font-weight: 500;
-    --#{$prefix}table-bg: #{$storm-grey-100};
-    padding: 8px;
+  thead {
+    td {
+      font-weight: 500;
+    }
   }
 
-  th, td {
-    border-bottom: 1px solid $storm-grey-300;
+  tr {
+    border-bottom: 1px solid #{$storm-grey-200};
```

Comment:
You should be able to override the `$table-border-color` to use `$storm-grey-200` in the `light.scss` and then you can change it here to use the `table-border-color`. it would allow us to just change things in the settings file and not have to edit the css to hunt down variables. :D